tachyon/crypto/commitments/pedersen/pedersen_unittest.cc

```diff
@@ -28,14 +29,15 @@ TEST_F(PedersenTest, CommitPedersen) {
       base::CreateVector(kMaxSize, []() { return math::bn254::Fr::Random(); });
 
   math::bn254::Fr r = math::bn254::Fr::Random();
-  math::bn254::G1JacobianPoint commitment;
+  math::bn254::G1AffinePoint commitment;
   ASSERT_TRUE(vcs.Commit(v, r, &commitment));
 
-  math::VariableBaseMSM<math::bn254::G1JacobianPoint> msm;
-  math::bn254::G1JacobianPoint msm_result;
-  ASSERT_TRUE(msm.Run(vcs.generators(), v, &msm_result));
+  math::VariableBaseMSM<math::bn254::G1AffinePoint> msm;
+  math::bn254::G1PointXYZZ msm_result_tmp;
+  ASSERT_TRUE(msm.Run(vcs.generators(), v, &msm_result_tmp));
+  math::bn254::G1AffinePoint msm_result = msm_result_tmp.ToAffine();
 
-  EXPECT_EQ(commitment, msm_result + r * vcs.h());
+  EXPECT_EQ(commitment, (msm_result + r * vcs.h()).ToAffine());
```

Comment:
Perhaps to name the variables more descriptively, `msm_result_xyzz` instead of `msm_result_tmp` and `men_result_affine` instead of `msm_result` would be better.  I also wanted to ask why `(msm_result + r * vcs.h()).ToAffine()` needs to be converted to affine if `msm_result` is already affine?

Reply:
Since `Affine` + `Affine` results in `Jacobian`, See the return type of `AffinePoint<>::Add(const AffinePoint&)` in `tachyon/math/elliptic_curves/short_weierstrass/affine_point.h`